docs/reference/api/docker_remote_api_v1.20.md

```diff
@@ -208,6 +209,7 @@ Json Parameters:
 -   **CpusetCpus** - String value containing the `cgroups CpusetCpus` to use.
 -   **CpusetMems** - Memory nodes (MEMs) in which to allow execution (0-3, 0,1). Only effective on NUMA systems.
 -   **BlkioWeight** - Block IO weight (relative weight) accepts a weight value between 10 and 1000.
+-   **MemorySwappiness** - Integer Value, Tuning Container memory swappiness. Accepts value between 0 and 100.
```

Comment:
@estesp Thanks for comments
1. changed to: Integer value, tuning container's memory swappiness
2. regarding Accepts values between 0 and 100: how about changing to
Accepts a value between 0 and 100, since we should be specifying only one value in that range at any point of time.